### ISSUE TITLE
Hyperic HQ 5.8.6 not support MySql？

### ISSUE THREAD

**Issue by zhangyy130** [hyperic/hq#18]
[hyperic@localhost hyperic-hq-installer-5.8.6]$ ./setup.sh -MySQL
Unpacking JRE to temporary directory /tmp/jre

MySQL
Please ignore references to missing tools.jar
Picked up _JAVA_OPTIONS: -Xmx768M
Unable to locate tools.jar. Expected to find it in /tmp/lib/tools.jar
Loading taskdefs...
Taskdefs loaded
Initializing Hyperic HQ 5.8.6 Installation...
Choose which software to install:
1: Hyperic HQ Server
2: Hyperic HQ Agent
You may enter multiple choices, separated by commas.
1
HQ server installation path [default '/home/hyperic']:

What port should the HQ server's web-based GUI listen on for http communication? [default '7080']:

What port should the HQ server's web-based GUI listen on for secure https communication? [default '7443']:

Choices:
        1: Yes
        2: No
Would you like to use your own java keystore? [default '2']:

Enter the base URL for the HQ server's web-based GUI [default 'http://localhost.ch:7080/']:
http://172.23.67.90:7080/
Enter the email address that HQ will use as the sender for email messages [default 'hqadmin@localhost.ch']:

Choices:
        1: HQ Built-in Database
        2: PostgreSQL
The HQ built-in database is provided for EVALUATION PURPOSES ONLY. For production purposes use vPosgreSQL. What backend database should the HQ server use? [default '1']:

**Comment by martinking1997** [comment]
No. Hyperic 5.8 does not support mysql to be backend db any longer.
